relnotes/verify.feature.md

```diff
@@ -0,0 +1,9 @@
+* `ocean.core.Verify`
+
+  New module with a single function, `verify`, intended to serve as drop-in
+  replacement for `assert` to comply to [Sociomantic assert/enforce
+  policies](https://github.com/sociomantic-tsunami/sociomantic/blob/master/Code/assert_vs_enforce.md)
```

Comment:
> Has this url been discussed?

I expected to get some suggestion how to name it from @dylan-cromwell-sociomantic but he merged it with this URL :P 

Adapting guideline is planned after this is merged.